File: Data Structures & Algorithms/concatenation-of-array/submission-0.java
class Solution {
    public int[] getConcatenation(int[] arr) {
        int n = arr.length;
        int[] ans = new int[n + n];

        for (int i = 0; i < n; i++) {
            ans[i] = arr[i];
            ans[i + n] = ans[i];
        }

        return ans;
    }
}

File: Data Structures & Algorithms/two-integer-sum/submission-0.py
class Solution:
    def twoSum(self, arr: List[int], target: int) -> List[int]:
        ans = []
        dic = {}

        for i in range(len(arr)):
            if target - arr[i] in dic:
                ans.append(dic.get(target - arr[i]))
                ans.append(i)
                break
            dic[arr[i]] = i
        
        return ans